include/grpcpp/impl/codegen/server_interface.h

```diff
@@ -185,13 +186,17 @@ class ServerInterface : public internal::CallHook {
           notification_cq_(notification_cq),
           tag_(tag),
           request_(request) {
-      IssueRequest(registered_method, &payload_, notification_cq);
+      IssueRequest(registered_method, payload_.c_buffer_ptr(), notification_cq);
```

Comment:
Can you change that to the polymorphic `bbuf_ptr` instead of `c_buffer_ptr` ? We should deprecate the latter soon, and the former allows us to define `IssueRequest` differently without additional changes.